runtime/src/iree-amd-aie/driver/xrt/api.h

```diff
@@ -16,6 +16,38 @@
 extern "C" {
 #endif  // __cplusplus
 
+//===----------------------------------------------------------------------===//
+// iree_hal_xrt_device_params_t
+//===----------------------------------------------------------------------===//
+
+typedef enum iree_hal_xrt_command_dispatch_type_e {
```

Comment:
Do we actually have these two modes? I think we can only do serialized dispatch in XRT?